examples/code/home/use-tools/call-tools-directly/math_sqrt_directly.py

```diff
@@ -1,13 +1,41 @@
 from arcadepy import Arcade
 
+# You can also set the `ARCADE_API_KEY` environment variable instead of passing it as a parameter.
 client = Arcade(api_key="arcade_api_key")
 
+# Arcade needs a unique identifier for your application user (this could be an email address, a UUID, etc).
+# In this example, simply use your email address as the user ID:
 user_id = "user@example.com"
 
+# Let's use the `Math.Sqrt` tool from the Arcade Math toolkit to get the square root of a number.
 response = client.tools.execute(
     tool_name="Math.Sqrt",
     input={"a": 625},
     user_id=user_id,
 )
 
+print(f"The square root of 625 is {response.output.value}")
+
+# Now, let's use a tool that requires authentication to star a GitHub repository
+
+auth_response = client.tools.authorize(
+  tool_name="GitHub.SetStarred",
+  user_id=user_id,
+)
+
+if auth_response.status != "completed":
+    print(f"Click this link to authorize: {auth_response.url}")
+    print("re-run the script after authorizing")
```

Comment:
Wondering if it would be better to introduce calling `client.auth.wait_for_completion` here? It feels like a nice way to have a quick "nice one!" early on.

Reply:
Great idea! a9f1dd57ea958880e238c382fcf37a1b226689eb